src/resmom/requests.c

```diff
@@ -2834,6 +2834,7 @@ req_cpyfile(struct batch_request *preq)
 	 * the same permissions as TMPDIR
 	 */
 	(void)revert_impersonated_user();
+	SetEnvironmentVariable("USER", NULL);
```

Comment:
not sure, but AFAIK, in code as of now I can see only two setenv for USER, and which is in fork_to_user() (both version windows as well as *nix), so my question is instead doing this can we remove that setenv from fork_to_user()?

Reply:
+1, going a bit further I think they were used in dce_login - refer to the deletions in the dce_login removal commit - https://github.com/openpbs/openpbs/commit/ad9f208faca09679082d889db56c677d8bfc7b85#diff-68d448a9a18cff40450417ba0ffc3e0dL280

Reply:
Hi @hirenvadalia, Thanks for review. I had this idea, but was not sure as pbs_idled search for that. And am not sure whether it break anything

Reply:
That pbs_idled.c is for *nix one, for windows there is pbs_idled_win.c which doesn't find user name, and in *nix pbs_idled is stared by user only (or login bashrc when user log-in into system) when cycle harvesting is configured, so even if we remove setenv from fork_to_user it won't affect pbs_idled.